### ISSUE TITLE
No such file or directory, mkdir 'pages/api/users/[id]'

### ISSUE THREAD

**Issue by nicholaschiang** [vinissimus/next-translate#235]
Right now, I'm using the dynamic routes alternative to the build step but I'm moving over to the build step and I encountered this error:
```
Building pages | from webpages to pages
🔨 / [ 'common', 'about', 'query' ]
(node:7356) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir 'pages/api/users/[id]'
    at Object.mkdirSync (fs.js:921:3)
    at copyFolderRecursiveSync (/home/nchiang/repos/tutorbook/node_modules/next-translate/cli/builder.js:217:8)
    at buildPageInAllLocales (/home/nchiang/repos/tutorbook/node_modules/next-translate/cli/builder.js:236:7)
    at /home/nchiang/repos/tutorbook/node_modules/next-translate/cli/builder.js:131:5
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

My `webpages` directory is as follows:
```
webpages
├── api
│   ├── account.ts
│   ├── appts
│   │   └── index.ts
│   ├── orgs
│   │   ├── [id].ts
│   │   └── index.ts
│   ├── README.md
│   ├── redirect.ts
│   └── users
│       ├── [id]
│       │   ├── index.ts
│       │   └── parents.ts
│       └── index.ts
├── _app.tsx
└── index.tsx

5 directories, 11 files
```
 And then the generated `pages` directory looks fine **except** for the `api/users/[id]` routes that gave the error:
```
pages
├── api
│   ├── account.ts
│   ├── appts
│   │   └── index.ts
│   ├── orgs
│   │   ├── [id].ts
│   │   └── index.ts
│   ├── README.md
│   ├── redirect.ts
│   └── users
│       └── index.ts
├── _app.tsx
└── index.js

4 directories, 9 files
```

**Comment by nicholaschiang** [comment]
At first I thought that this might perhaps be a problem with the dynamic `[id]` directory name, but it's not. This still occurs when I change the name from `[id]` to `id`:
```
Building pages | from webpages to pages
🔨 / [ 'common', 'about', 'query' ]
(node:7831) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir 'pages/api/users/id'
    at Object.mkdirSync (fs.js:921:3)
    at copyFolderRecursiveSync (/home/nchiang/repos/tutorbook/node_modules/next-translate/cli/builder.js:217:8)
    at buildPageInAllLocales (/home/nchiang/repos/tutorbook/node_modules/next-translate/cli/builder.js:236:7)
    at /home/nchiang/repos/tutorbook/node_modules/next-translate/cli/builder.js:131:5
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

**Comment by aralroca** [comment]
@nicholaschiang thanks for reporting this issue! I'm going to take a look

**Comment by aralroca** [comment]
I tried to reproduce it but it seems that it works well in my case. Would be great if you can share a reproducible example 👏

**Comment by aralroca** [comment]
In 1.0.0 we are going to change the build step by a Next.js plugin (using a custom webpack loader), I prereleased an experimental version under 1.0.0-experimental.1. This means that after 1.0.0 this will be automatically fixed

**Comment by aralroca** [comment]
Closing this. Already fixed on [1.0.0-canary.1](https://github.com/vinissimus/next-translate/releases/tag/1.0.0-canary.1). The 1.0 will be released in approximately 1-2 weeks. 😊
